cirrus/server/tests/conftest.py

```diff
@@ -51,3 +52,114 @@ def fml():
 @fixture
 def exception():
     return Exception("some error")
+
+
+@fixture(name="reset_glean", scope="function", autouse=True)
+def fixture_reset_glean():
+    testing.reset_glean(application_id="cirrus-test", application_version="0.1.0")
+
+
+@fixture
+def get_telemetry():
+    return initialize_glean()
```

Comment:
Resetting glean via the reset API also initializes Glean, so initializing it before a test could be accomplished by just calling that API. There is also an [optional parameter](https://github.com/mozilla/glean/blob/4198786204fa19c041bdefe7abb91ddc193747db/glean-core/python/glean/testing/__init__.py#L27) to the reset function if you need to re-initialize Glean without clearing the stored information.